portfolio/src/main/webapp/scripts/generate_content.js

```diff
@@ -0,0 +1,50 @@
+import { ICONS, CONTENTS, GOOGLE_COLORS, TAG_COLORS, HEADERS } from './constants.js';
+
+const containerIds = ICONS.slice(1); // removes Contact
+
+const createTag = (content, color) => {
```

Comment:
backgroundColor -- to avoid confusion with color below (and color CSS attribute in general)